src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/SPDXExtensions.cs

```diff
@@ -106,7 +106,13 @@ public static void AddSpdxId(this CreationInfo creationInfo)
 
     public static void AddSpdxId(this Common.Spdx30Entities.Relationship relationship)
     {
-        relationship.SpdxId = GenerateSpdxIdBasedOnElement(relationship, relationship.To + relationship.RelationshipType.ToString());
+        var relationsipToString = string.Empty;
```

Comment:
Typo? Will need to be fixed in multiple lines
```suggestion
        var relationshipToString = string.Empty;
```